solidity/security/thirdweb-vulnerability.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+- id: thirdweb-vulnerability
+  pattern-either:
+    - pattern-regex: 'import ".*\/(.*)(e|E)(r|R)(c|C)2771(.*)\.sol";((.|\n)*)import ".*\/(.*)(m|M)(u|U)(l|L)(t|T)(i|I)(c|C)(a|A)(l|L)(l|L)(.*)\.sol";'
```

Comment:
pattern-regex is PCRE compatible, you can use regex modifiers, use `(?i)` for case insensitive matching: ```- pattern-regex: '(?i)import...```